subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformCachingIntegrationTest.groovy

```diff
@@ -73,6 +73,29 @@ class ArtifactTransformCachingIntegrationTest extends AbstractHttpDependencyReso
         output.count("Transformed") == 0
     }
 
+    def "task cannot write into transform directory"() {
+        def forbiddenPath = ".transforms/not-allowed.txt"
+
+        buildFile << """
+            subprojects {
+                task badTask {
+                    outputs.file { project.layout.buildDirectory.file("${forbiddenPath}") }
+                    doLast {
+                        project.layout.buildDirectory.file("${forbiddenPath}").get().asFile.text = "notAllowed"                        
+                    }
+                }
+            }
+        """
+
+        when:
+        fails "badTask", "--continue"
```

Comment:
Why do we need `--continue`?

Reply:
I want to see all the task failures, not only the first one.